src/entity/Unit.ts

```diff
@@ -1151,13 +1167,13 @@ export function isBoss(unitSourceId: string) {
 }
 
 // Returns whether or not a unit is truly dead
-// Considers game state and soulshard modifier
-// so that soul shard owners are considered "remaining"
-// since they will be resurrected on their next turn
```

Comment:
Why remove the soul shard owner comment?

Reply:
I replaced it with:
```ts
// Considers game state and undying effects
// to-be-revived enemies are counted in "remaining" units
```
...which was meant to be more generic / encapsulate undying, soul shard, and any future modifiers that would cause a unit to be resurrected. I try to make the comment more clear.